src/Http/Requests/QuestionCreateRequest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace EscolaLms\Questionnaire\Http\Requests;
+
+use EscolaLms\Questionnaire\Models\Question;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Gate;
+
+class QuestionCreateRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return Gate::allows('create', Question::class);
+    }
+
+    public function rules(): array
+    {
+        return [
+            'title' => 'string|required',
+            'description' => 'string|required',
+            'questionnaire_id' => 'integer|required',
```

Comment:
https://laravel.com/docs/8.x/validation#rule-exists

```suggestion
            'questionnaire_id' => 'integer|required|exists:questionnares',
```

Reply:
done